mne/report/report.py

```diff
@@ -278,6 +279,41 @@ class _ContentElement:
     html: str
 
 
+def _check_tags(tags) -> Tuple[str]:
+    # Must be iterable, but not a string
+    if (isinstance(tags, str) or not isinstance(tags, (Sequence, np.ndarray))):
+        raise TypeError(
+            f'tags must be a collection of str, but got {type(tags)} '
+            f'instead: {tags}'
+        )
+    tags = tuple(tags)
+
+    # Check for invalid dtypes
+    bad_tags = [tag for tag in tags
+                if not isinstance(tag, str)]
+    if bad_tags:
+        raise TypeError(
+            f'tags must be strings, but got the following instead: '
+            f'{", ".join([str(tag) for tag in bad_tags])}'
+        )
+
+    # Check for invalid characters
+    invalid_chars = (' ', '"', '\n')  # we'll probably find more :-)
+    bad_tags = []
+    for tag in tags:
+        for invalid_char in invalid_chars:
+            if invalid_char in tag:
+                bad_tags.append(tag)
+                break
+    if bad_tags:
+        raise ValueError(
+            f'The following tags contained invalid characters: '
+            f'{", ".join(bad_tags)}'
```

Comment:
This will add quotes (and probably escape `"`) which will make it clearer I think
```suggestion
            f'{", ".join(repr(tag) for tag in bad_tags)}'
```